core/src/java/com/pb/mtctm2/abm/accessibilities/BestTransitPathCalculator.java

```diff
@@ -736,9 +351,9 @@ public float calcPathUtility(TransitWalkAccessDMU walkDmu, TransitDriveAccessDMU
      * @param departPeriod Departure time period - 1 = AM period, 2 = PM period, 3 =OffPeak period
      * @param debug boolean flag to indicate if debugging reports should be logged
      * @param logger Logger to which debugging reports should be logged if debug is true
-     * @return double[][] Array of best tap pair values - rows are N-path, columns are orig tap, dest tap, skim set, utility
+     * @return double transit utility
      */
-    public double[][] getBestTapPairs(TransitWalkAccessDMU walkDmu, TransitDriveAccessDMU driveDmu, int accMode, int origMgra, int destMgra, int departPeriod, boolean debug, Logger myLogger, float odDistance)
+    public double calcPersonSpecificUtilities(int pTaz, int aTaz, TransitWalkAccessDMU walkDmu, TransitDriveAccessDMU driveDmu, int accMode, int origMgra, int destMgra, int departPeriod, boolean debug, Logger myLogger, float odDistance)
```

Comment:
@lmz, @AshishKuls, @i-am-sijia 

Per our call, a better approach would be to:

1. Start with a walk and drive specific versions of the `calcPathUtility` method. The methods would have the same name, but one would expect a walkDMU and the other a driveDMU.
2. Propagate this pattern up to this method, again creating two versions, one for walk and one for drive.  

When invoking these methods when accessing walk paths, the code would therefore not need the drive details. 

Given our current focus on getting TM2.2.1 up and running and the likelihood that making this change will likely break something else, we will note this for now but move forward.